src/MakeListingDisplay/components/MakeListingDisplay.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+
+import InputField from './InputField';
+
+import '../styles/MakeListingDisplay.css';
+
+/**Component class that displays the input fields for a user to post a listing */
+class MakeListingDisplay extends Component {
+    render(){
+        return (
+            <div>
+                <div id="header">
+                    <h1>Post a Listing</h1>
+                </div>
+                <form action="/listings" id="listingForm" method="POST">
+                    <InputField fieldHeader="Title:" fieldName="title" fieldType="text" />
+                    <InputField fieldHeader="Description:" fieldName="description" fieldType="text" />
+                    <InputField fieldHeader="Total number of rooms in apartment:" fieldName="numRooms" fieldType="number" />
+                    
+                    <p>Number of rooms for rent:</p>
+                    <div id="singleInfo">
+                        <InputField fieldHeader="Singles:" fieldName="numSingles" fieldType="number" />
+                        <InputField fieldHeader="Monthly Rent Per Single:" fieldName="singlePrice" fieldType="number" />
+                    </div>
+                    <div id="sharedInfo">
+                        <InputField fieldHeader="Shared:" fieldName="numShared" fieldType="number" />
+                        <InputField fieldHeader="Monthly Rent Per Shared Room:" fieldName="sharedPrice" fieldType="number" />
+                    </div>
+                    <div id="listingPriceDisplay"></div>
+
+                    <InputField fieldHeader="" fieldName="listingPrice" fieldType="hidden" />
```

Comment:
Are you planning on implementing this listing price calculation later?

Reply:
Yeah that's part of the next pr